app-web/topicRegistry/platform-service-migration.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Platform Service Migration",
+  "description": "A series of migration plan for the platform services provided",
+  "attributes": {
+    "labels": [
```

Comment:
labels are key value pairs and not an array just like openshift.

Reply:
not sure what you mean...

Reply:
```js
labels: {
  ministry: 'foo',
}
```